docs/usage.md

```diff
@@ -0,0 +1,86 @@
+# Usage
+
+Use the import tool from the command line, like so:
+```
+java -jar <path-to-jar> <mode> <list of mcf/tmcf/csv files>
+```
+
+Hint: it can be useful to create an alias for the jar file, such as:
+```
+alias dc-import='java -jar <path-to-jar>'
+```
+
+This is the form that will be used in the rest of the documentation.
+
+Hint: to access a concise explanation of usage modes and flags, run
+`dc-import --help`
+
+## Usage Modes
+
+The import tool can either lint StatVars and other schema files, or
+it can generate instance MCFs from a TMCF and CSV.
+
+### Lint Mode (`lint`)
+To run the tool in lint mode, use:
+```
+dc-import lint <list of mcf files>
+```
+
+### Instance MCF Generation Mode (`genmcf`)
+To run the tool in genmcf mode, use:
+```
+dc-import genmcf <list of csv/tmcf files>
+```
+
+Note that in this mode, schema file(s) are optional to pass in.
+
+## Flags
+Flags available for the tool are listed below.
+
+You can also run `dc-import --help` to see a list of flags in your terminal.
+### `-e`, `--existence-checks`
+Check DCID references to schema nodes against the KG and locally. If set, then
+calls will be made to the Staging API server, and instance MCFs get fully loaded
+into memory. 
+
+Defaults to true.
+
+### `-h`, `--help`
+Show a help message and exit.
+
+### `-n`, `--num-threads=<numThreads>`
+Number of concurrent threads used for processing CSVs.
+
+Defaults to 1.
+
+### `-o`, `--output-dir=<outputDir>`
+Directory to write output files.
+
+Default is `dc_generated/` within current working directory.
+
+### `-s`, `--stat-checks`
+Checks integrity of time series by checking for holes, variance in values, etc.
+
+Defaults to true.
+
+### `-p`, `--sample-places=<samplePlaces>`
+List of place dcids to run stats check on.
+
+This should only be set if `--stat-checks` is true. If `--stat-checks` is true and this is not set, 5 sample places are picked for roughly each distinct place type.
+
+### `-r`, `--resolution=<resolutionMode>`
+Specifies the mode of resolution to use: `NONE`, `LOCAL`, `FULL`.  For no resolution, set NONE.  To lookup external IDs (like ISO) in DC, resolve local references and generated DCIDs, set FULL. To just resolve local references and generate DCIDs, set LOCAL.  Note that FULL mode may be slower since it makes (batched) DC Recon API calls and two passes over your CSV files.
```

Comment:
Lets enumerate as a list, and for each, document behavior?

* `NONE`:  No resolution
* `LOCAL`: ...
* `FULL`: ...